src/DataContainer/PaletteManipulator.php

```diff
@@ -87,6 +92,22 @@ public function addField($name, $parent, string $position = self::POSITION_AFTER
         return $this;
     }
 
+    /**
+     * If $parent is empty the field is removed everythere.
```

Comment:
`everywhere`. Also, can you add a description on what `$parent` should represent?

Reply:
Done with https://github.com/contao/core-bundle/pull/1668/commits/ea46a4996a5b91fd5d9c963608bade3021f91341